src/test/kotlin/org/rust/ide/highlight/RsRainbowVisitorTest.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.highlight
+
+import org.intellij.lang.annotations.Language
+import org.rust.lang.RsTestBase
+import org.rust.lang.core.psi.RsPatBinding
+import org.rust.lang.core.psi.RsPath
+
+class RsRainbowVisitorTest : RsTestBase() {
+    fun checkRainbow(@Language("Rust") text: String) {
+        InlineFile(text)
```

Comment:
Looks like `testRainbow` does `configureByFile` internally, so `InlineFile` does not have any effect.